cluster-operator/src/test/java/io/strimzi/operator/cluster/model/ModelUtilsTest.java

```diff
@@ -111,7 +112,7 @@ public void testParsePodDisruptionBudgetTemplate()  {
                 .withMaxUnavailable(2)
                 .build();
 
-        Model model = new Model(kafka);
+        Model model = new Model(new Reconciliation("test", kafka.getKind(), kafka.getMetadata().getNamespace(), kafka.getMetadata().getName()), kafka);
```

Comment:
Any reason this does not use DUMMY_RECONCILIATION? Same also for the other cases in this file.